rspec-retry.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |gem|
   gem.name          = "rspec-retry"
   gem.require_paths = ["lib"]
   gem.version       = RSpec::Retry::VERSION
-  gem.add_runtime_dependency(%{rspec-core}, '>3.3', '<=3.5')
+  gem.add_runtime_dependency(%{rspec-core}, '>3.3', '<3.6')
```

Comment:
it's definitely tested for 3.3, 3.4, and 3.5, that's why we constrain the max version. But this is just better. Will release a new version with the change. Thanks!